src/test/java/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParseTreeTest.java

```diff
@@ -78,6 +81,10 @@ private static String getDocPath(String filename) throws IOException {
         return getPath("javadocTags" + File.separator + filename);
     }
 
+    private static String getOutputDocPath(String filename) throws IOException {
+        return getDocPath("output" + File.separator + filename);
```

Comment:
@Vladlis Folder names should be how we define our standard pattern in https://github.com/checkstyle/checkstyle/issues/3891 .
In `AstRegressionTest`, as a similar example, we name text tree files with the postfix `Ast.txt`.